server/DataStorageController.js

```diff
@@ -114,12 +114,33 @@ function handleEmptyDBError () {
 	sqldumpImporter()
 }
 
+let ASKED_QUESTIONS = {
+	1: [],
+	2: [],
+	3: [],
+	4: []
+}
+
 // Exports --------------------------------------------------------------------
 module.exports.readAQuestion = async function (categoryID) {
```

Comment:
I think the app is using the getRandomQuestionByCategory/getQuestionsByCategory because it is searching based on the category name and not he category id that is in the db. Either this need to be modified for the getRandomQuestionByCategory, or the app needs a modification to successfully call this function